Admin/AclAdminExtension.php

```diff
@@ -21,6 +21,8 @@
 use Symfony\Component\Security\Core\Role\RoleHierarchy;
 use Symfony\Component\Security\Core\Role\RoleInterface;
 use Symfony\Component\Security\Core\SecurityContextInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Acl\Domain\UserSecurityIdentity;
```

Comment:
> +use Symfony\Component\Security\Acl\Domain\UserSecurityIdentity;

removed

> +use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;

-- @meyerbaptiste if I remove it, won't be able to check in code:

`if (!$tokenStorage instanceof TokenStorageInterface `